build-scripts/travis.sh

```diff
@@ -401,6 +405,21 @@ build_auth() {
   run "find /tmp/pdns-install-dir -ls"
 }
 
+build_ixfrdist() {
+  run "autoreconf -vi"
+  run "./configure \
+    ${sanitizerflags} \
```

Comment:
Perhaps it would be nice to pass at least `--enable-asan` when building `ixfrdist`?